browser-test/src/support/admin_programs.ts

```diff
@@ -720,42 +730,42 @@ export class AdminPrograms {
     blockDescription = 'screen description',
     questionNames: string[] = [],
   ) {
-    const questionSpecs: QuestionSpec[] = questionNames.map((qName) => {
-      const questionSpec: QuestionSpec = {name: qName, isOptional: false}
-      return questionSpec
+    return await this.addProgramBlockUsingSpec({
+      programSelector: {name: programName},
+      block: {
+        description: blockDescription,
+        questions: questionNames.map((questionName) => ({
+          name: questionName,
+        })),
+      },
     })
-    return await this.addProgramBlockUsingSpec(
-      programName,
-      blockDescription,
-      questionSpecs,
-    )
   }
 
   /**
-   * Creates a new program block with the given questions as defined by {@link QuestionSpec}.
+   * Creates a new program block as defined by {@link ProgramBlockSpec}.
    *
-   * Prefer this method over {@link #addProgramBlock}: This method provides the same functionality
-   * but also makes it easy to use optional questions.
+   * Prefer this method over {@link #addProgramBlock}.
    */
-  async addProgramBlockUsingSpec(
-    programName: string,
-    blockDescription = 'screen description',
-    questions: QuestionSpec[] = [],
-    isProgramDisabled: boolean = false,
-  ) {
-    await this.gotoEditDraftProgramPage(programName, isProgramDisabled)
+  async addProgramBlockUsingSpec(spec: ProgramBlockSpec) {
```

Comment:
```suggestion
  async addProgramBlockUsingSpec(programBlockSpec: ProgramBlockSpec) {
```
(blocking) Since we now have multiple interfaces with spec in the name let's write this out in full to avoid confusion

Reply:
Happy to make this change - can you help me understand where it will add clarity? Where would one see the parameter name without the type that would be unclear now?

- When writing code that calls addProgramBlockUsingSpec, the IDE will show me both the name and type of the param
- In this method there's only one object called spec - I guess it would be clearer there to call it programBlockSpec?

Are there other cases I'm missing?

Reply:
In a code review on github it defaults to showing only a few changed lines. With a more descriptive name reviewing it will be easier otherwise you'll have to click the show more button until you get to the method definition.

The same if looking at code on github itself.

Not all editors work the same showing the same information.

Reply:
Thanks for explaining! This is moot with the latest changes.